catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -347,12 +347,18 @@ public void init() {
                     .map(Result::getMetacard)
                     .map(metacard -> {
                         boolean isSubscribed = ids.contains(metacard.getId());
-
-                        return ImmutableMap.builder()
-                                .putAll(transformer.transform(metacard))
-                                .put("subscribed", isSubscribed)
-                                .build();
+                        try {
+                            return ImmutableMap.builder()
+                                    .putAll(transformer.transform(metacard))
+                                    .put("subscribed", isSubscribed)
+                                    .build();
+                        } catch (RuntimeException e) {
+                            LOGGER.warn(
```

Comment:
This means that a third party system is sending bad data, and an administrator **should** be notifed.